connection.go

```diff
@@ -755,6 +756,18 @@ func (sc *snowflakeConn) buildRowsForRunningQuery(ctx context.Context, qid strin
 	return rows, err
 }
 
+func (sc *snowflakeConn) setupOCSPPrivatelink(app string, host string) error {
+	ocspCacheServer := fmt.Sprintf("http://ocsp.%v/ocsp_response_cache.json", host)
+	if err := os.Setenv(cacheServerURLEnv, ocspCacheServer); err != nil {
+		return err
+	}
+	ocspRetryHost := fmt.Sprintf("http://ocsp.%v/retry/", host) + "%v/%v"
```

Comment:
do you need some kind of escaping here for `"%v/%v"` ?

Reply:
Nope! It's treated as a literal there.